python/paddle/v2/dataset/common.py

```diff
@@ -201,8 +201,7 @@ def close_writers(w):
     def write_data(w, lines):
         random.shuffle(lines)
         for i, d in enumerate(lines):
-            d = pickle.dumps(d, pickle.HIGHEST_PROTOCOL)
-            w[i % num_shards].write(d)
+            w[i % num_shards].write(str(d))
```

Comment:
Curious why we need pickle? since the master is parsing recordio files directly.